README.rst

```diff
@@ -9,6 +9,16 @@ Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/about>`_.
 Currently, Windows PowerShell, Command Prompt, and Git Shell for
 Windows are supported with limitations (See `Windows Issues`_).
 
+This product is supported by the Cybersecurity Development team at the 
+University of Illinois, on a best-effort basis. As of the last update to 
+this README, the expected End-of-Life and End-of-Support dates of this 
```

Comment:
What is the difference between End-of-Life and End-of-Support? Do we have a standard definition for our team?

Reply:
See https://github.com/techservicesillinois/secdev-job-aids/issues/2#issuecomment-797049917

Reply:
@ddriddle  We have a U of I Technology Services Knowledge Base article in draft that will publicly define the difference as we use the terms.

For our purposes on this effort, they are identical. 

We decided to include both terms here so that anyone looking for a more nuanced answer doesn't need to reach out to ask which we mean.